pkg/cli/cvd.go

```diff
@@ -250,7 +251,7 @@ func (c *cvdCreator) createCVDFromLocalBuild() ([]*hoapi.CVD, error) {
 		return nil, err
 	}
 	req := &hoapi.CreateCVDRequest{EnvConfig: envConfig}
-	res, err := hostSrv.CreateCVD(req, c.credentialsFactory())
+	res, err := hostSrv.CreateCVD(req, c.credentialsFactory(), c.opts.ProjectID)
```

Comment:
Both the credential string and the project ID are used when calling the build APIs. Can we define a struct to put them together and let `type CredentialsFactory func()` returns this struct?